src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,32 @@
+import Popup from "../components/Popup.js";
+
+export class PopupWithConfirmation extends Popup {
+  constructor(popup, { handleFormSubmit, renderLoading }) {
+    super(popup);
+    this.handleFormSubmit = handleFormSubmit;
+    this.form = this.popup.querySelector(".form");
+
+    this._renderLoading = renderLoading;
+  }
+  close() {
+    super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.